### PR TITLE
Add CryptoX to documentation

### PR DESCRIPTION
**Pull request by dg-concordium**
Add CryptoX to documentation

Many topics touched by this.
Original structure is not changed as this will be part of a later update.

@Radiokot: Please review that this is technically correct.

### CODE REVIEW

**Comment by Radiokot** [comment]
The preview link can't be trusted. Reviewers must check out `add--cryptox` branch and run the site locally

File: source/mainnet/net/guides/tokens.rst [hunk 1]
```diff
@@ -5,7 +5,96 @@
...
+
+            .. image:: ../images/cryptox/cryptox-look-for-tokens.png
+                :width: 50%
+
+        #. Select the token(s) you want to add. You can tap on a token to see more information about it and then tap **Back to list** to return to the list. Tap **Apply** once you have made your selection(s).
+
+            .. image:: ../images/cryptox/cryptox-select-token.png
+                :width: 50%
+
+        If you have purchased tokens on a marketplace, it can suggest tokens to be added directly without going through the steps above. In this case, the screen below appears when your wallet is connected and the service suggests the tokens to be added. Tap **Add tokens** to add the selected tokens to your wallet.
```
Comment (Radiokot) [inline source/mainnet/net/guides/tokens.rst:31]: Not possible in CryptoX

File: source/mainnet/net/guides/tokens.rst [hunk 2]
```diff
@@ -5,7 +5,96 @@
...
+        #. Add the receiver address.
+
+        #. Tap **Continue**.
+
+        #. Review the transaction. When satisfied, tap **Send funds**.
+
+            .. image:: ../images/cryptox/cryptox-send-tokens2.png
+                :width: 50%
+
+        #.Enter your passcode.
```
Comment (Radiokot) [inline source/mainnet/net/guides/tokens.rst:76]:
```suggestion
        #. Enter your passcode.
```

File: source/mainnet/net/installation/downloads.rst
Comment (Radiokot) [inline source/mainnet/net/installation/downloads.rst:1]: Let's also add a link to GitHub, where standalone installable packages (APKs) for Android can be downloaded
https://github.com/Concordium/cryptox-android/releases

**Review by Radiokot — CHANGES_REQUESTED** [review]
I left some comments

File: source/mainnet/net/browser-wallet/connect-app.rst [hunk 3]
```diff
@@ -5,7 +5,31 @@
 Connect dApps to wallets
 ===================================
 
-You can connect the |bw| and |mw-gen2| to a `dApp <https://en.wikipedia.org/wiki/Decentralized_application>`__ that has a frontend interface so that you can pay for services. You can initiate the request from within the |bw| or |mw-gen2|, or the dApp can initiate a connection request that you must confirm. Connection can be made by either scanning a QR code or from a link to the dApp service.
+You can connect the |cryptox|, |bw|, and |mw-gen2| to a `dApp <https://en.wikipedia.org/wiki/Decentralized_application>`__ that has a frontend interface so that you can pay for services. You can initiate the request from within the |bw| or |mw-gen2|, or the dApp can initiate a connection request that you must confirm. Connection can be made by either scanning a QR code or from a link to the dApp service.
```
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/browser-wallet/connect-app.rst:8]: I have made the suggested change.

File: source/mainnet/net/concepts/concepts-transactions.rst [hunk 4]
```diff
@@ -9,36 +9,39 @@ A transaction on the Concordium blockchain is an operation which applies some ch
...
+
++----------------------------+---------------+----------------+-------------------+---------------+-------------------+-----------+
+|                            | |mw-gen1|     | |mw-gen2|      | |bw|              |Desktop Wallet | Concordium Client | |cryptox| |
++============================+===============+================+===================+===============+===================+===========+
+| Send CCD                   | |check|       | |check|        | |check|           | |check|       | |check|           ||check|    |
++----------------------------+---------------+----------------+-------------------+---------------+-------------------+-----------+
+| Send CCD with a schedule   |               |                |                   | |check|       | |check|           |           |
++----------------------------+---------------+----------------+-------------------+---------------+-------------------+-----------+
+| Unshield CCD               | |check|       | |check|        |                   | |check|       | |check|           |           |
```
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/concepts/concepts-transactions.rst:21]: Fixed.

File: source/mainnet/net/concepts/concepts-transactions.rst [hunk 5]
```diff
@@ -9,36 +9,39 @@ A transaction on the Concordium blockchain is an operation which applies some ch
...
+| Connect to dApps           |               | |check|        | |check|           |               |                   | |check|   |
++----------------------------+---------------+----------------+-------------------+---------------+-------------------+-----------+
+| Delegation                 | |check|       | |check|        | |check|           | |check|       | |check|           | |check|   |
++----------------------------+---------------+----------------+-------------------+---------------+-------------------+-----------+
+| Smart contract transactions|               |                |                   |               | |check|           |           |
++----------------------------+---------------+----------------+-------------------+---------------+-------------------+-----------+
+| Manage tokens              |               | |check|        |                   | |check|       + |check|           + |check|   +
++----------------------------+---------------+----------------+-------------------+---------------+-------------------+-----------+
+
+- |cryptox|: supports send CCD, receive CCD, validator management, delegation, connect to dApps, add and manage tokens.
```
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/concepts/concepts-transactions.rst:44]: Fixed

File: source/mainnet/net/guides/cryptox-faq.rst [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+
+.. dropdown:: Can I access my wallet on multiple devices with the seed phrase?
+
+    Yes, you can access your wallet concurrently using the same secret recovery phrase with the |cryptox|, |mw-gen2| and |bw|.
+
+    Be aware that any names you have given to identities and accounts are **specific to the device** so they will not appear when you recover the wallet on another device.
+
+    Also note that addresses are specific to the device. Only addresses for your own account are added to the address book on the new device, other addresses must be added manually.
+
+    It is also important to note that if you recover a wallet on two devices from the same seed phrase, only account balances are synchronized between the two devices.
```
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/guides/cryptox-faq.rst:80]: I have made the suggested change.

File: source/mainnet/net/guides/tokens.rst (hunk 1 above)
Comment (TinaKT) [reply source/mainnet/net/guides/tokens.rst:31]: I have deleted the section.

File: source/mainnet/net/installation/downloads.rst
Comment (TinaKT) [reply source/mainnet/net/installation/downloads.rst:1]: Link added

File: source/mainnet/net/guides/deciding-wallet.rst
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/guides/deciding-wallet.rst:1]: I agree, I have re-written the whole page.

File: source/mainnet/net/guides/shield-ccd-wallets.rst
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/guides/shield-ccd-wallets.rst:1]: I have removed gen-1 and gen-2 wallets from this section.

File: source/mainnet/net/guides/recover-wallet.rst [hunk 7]
```diff
@@ -5,15 +5,124 @@
...
+
+    #. Your wallet has now been restored on |cryptox|.
+
+        .. image:: ../images/cryptox/cryptox-recover-wallet9.png
+            :width: 40%
+
+
+    .. Note::
+
+        When you recover your wallet, any account names that you might have edited will be reset to the account number. You can :ref:`edit the account name<change-mw-acct-name>`, if desired.
```
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/guides/recover-wallet.rst:77]: Fixed.

**Review by Radiokot — APPROVED** [review]
Well done 👍🏻

File: source/mainnet/net/snippets/delegation-faq.rst [hunk 8]
```diff
@@ -41,7 +50,8 @@ For delegators
 
 .. dropdown:: Is there a “cool-down” period when delegating and how does it work?
 
-    Yes, for certain changes there is a :term:`cool-down<cool-down period>` period built into the pool system. The cool-down period is three weeks. During the cool-down period it is not possible to change the delegated amount or move the delegated amount to a different staking pool. The cool-down period is activated when you decrease the delegated amount or stop delegation entirely.
+    If you decrease or remove delegations, the funds are not released to your account until after a :term:`cool-down<cool-down period>` period.
+    The cool-down period is three weeks. The cool-down period is activated when you decrease the delegated amount or stop delegation entirely.
```
Comment (schorle1) [inline source/mainnet/net/snippets/delegation-faq.rst:54]: Is this in line with P7 updates?

File: source/mainnet/net/resources/glossary.rst [hunk 9]
```diff
@@ -340,9 +344,13 @@ Also see the Concordium `Whitepaper <https://developer.concordium.software/gover
 
       Used for smart contracts. A description of how to represent bytes in a more structured representation. It can be used by external tools when displaying the return value of a receive function and for specifying parameters using a structured representation, such as JSON. This makes it more human readable.
 
+   Seed phrase
+
+      A group of random words generated by the wallet that allows you to access the CCDs stored in it across devices or in case of non-functioning devices. Secret recovery phrase is supported by |mw-gen2| and |cryptox|.
+
    Secret recovery phrase
```
Comment (schorle1) [inline source/mainnet/net/resources/glossary.rst:351]: seed phrase should be the term we use

**Review by schorle1 — COMMENTED** [review]
Done :)

File: source/mainnet/net/concepts/concepts-transactions.rst [hunk 10]
```diff
@@ -66,6 +69,11 @@ You can see the fee in the transaction log.
 Tools
 =====
 
+The |cryptox|
+------------------
+
+The |cryptox| is a digital smartphone wallet with a simpler onboarding process that enables you to create and manage your Concordium identities and accounts, to create simple and shielded transactions, validate and delegate, add and manage tokens, connect to dApps, and to export and import your accounts and identities.
```
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/concepts/concepts-transactions.rst:75]: I agree - fixed.

File: source/mainnet/net/guides/cryptox-faq.rst [hunk 11]
```diff
@@ -0,0 +1,94 @@
...
+
+.. dropdown:: How is a seed phrase different from private keys?
+
+    :term:`Private keys` allow you to send, spend, and delegate your CCDs.
+
+    Your :term:`seed phrase` gives you access to your wallet and all of the private keys in the wallet. You can think of a wallet as being like a password manager for your accounts. As long as you have your master password (the seed phrase), you have access to all CCDs in the wallet.
+
+.. dropdown:: What happens if I lose my seed phrase?
+
+    If you accidentally throw away the paper your :term:`seed phrase` is written on, forget where you hid it, or do not pass it on to an heir, you no longer have the ability to :ref:`recover your wallet<recover-wallet>` and can lose access to your CCDs. If someone steals your seed phrase, they can access your CCDs. **If you lose your secret recovery phrase you lose access to your CCDs.**
```
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/guides/cryptox-faq.rst:32]: Fixed

File: source/mainnet/net/guides/cryptox-faq.rst [hunk 12]
```diff
@@ -0,0 +1,94 @@
...
+    Your validator status and staking pool participation will remain unchanged after the migration. Your rewards will continue to be distributed as usual.
+
+.. dropdown:: If I am awaiting scheduled transfers in an account in |mw-gen1| how should I migrate to |cryptox|?
+
+    Just download |cryptox| and :ref:`recover your wallet<recover-wallet>` with your backup file and password.
+    All of your scheduled transfers, along with your accounts and identities, will be transferred to your new |cryptox|.
+
+.. dropdown:: Can I access my wallet on multiple devices with the seed phrase?
+
+    Yes, you can access your wallet concurrently using the same secret recovery phrase with the |cryptox|, |mw-gen2| and |bw|.
```
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/guides/cryptox-faq.rst:74]: Fixed

File: source/mainnet/net/resources/glossary.rst (hunk 9 above)
Comment (TinaKT) [reply source/mainnet/net/resources/glossary.rst:351]: Fixed

File: source/mainnet/net/guides/cryptox-faq.rst (hunk 11 above)
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/guides/cryptox-faq.rst:32]: lose is correct. No changes.

File: source/mainnet/net/snippets/delegation-faq.rst (hunk 8 above)
Comment (TinaKT) [reply source/mainnet/net/snippets/delegation-faq.rst:54]: Documentation updates relating to P7, including this, have been reviewed by Thomas so it should be.

File: source/mainnet/net/guides/cryptox-faq.rst [hunk 13]
```diff
@@ -0,0 +1,94 @@
...
+
+.. dropdown:: What is a seed phrase?
+
+    A :term:`seed phrase` is a “master key” that unlocks all of your Concordium accounts. When entered into the wallet in the correct order, the 24 words in the seed phrase will recover all of the private keys you were storing on your original wallet and give access to all :term:`CCDs<ccd>` in the wallet. This means that even if you lose your physical hardware device, you’ll still have access to your blockchain assets. Seed phrases are sometimes referred to as secret recovery phrases, mnemonic phrases, mnemonic seeds, and backup phrases.
+
+.. dropdown:: How is a seed phrase different from private keys?
+
+    :term:`Private keys` allow you to send, spend, and delegate your CCDs.
+
+    Your :term:`seed phrase` gives you access to your wallet and all of the private keys in the wallet. You can think of a wallet as being like a password manager for your accounts. As long as you have your master password (the seed phrase), you have access to all CCDs in the wallet.
```
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/guides/cryptox-faq.rst]: Yes - fixed.

File: source/mainnet/net/guides/cryptox-faq.rst (hunk 11 above)
[the remark this answers is not in the document]
Comment (TinaKT) [reply source/mainnet/net/guides/cryptox-faq.rst]: Fixed.

**Pull request merged**